### PR TITLE
Create claws-mail.md

### PR DESCRIPTION
**Pull request by beli3ver**
Create HowTo to use vdirsyncer with Claws-Mail

### CODE REVIEW

File: docs/claws-mail.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Then search for the for the vCard in the folder **~/.contacts/**.
+Click ok, and you we will see your conatcts.
+
+**Important:** Claws-Mail shows only the contacts with a mail address.
+
+## Crontab
+At the end we create a crontab, so that vdirsyncer syncs automatically every 30 minutes our contacts:
+
+    contrab -e
```
Comment (untitaker) [inline docs/claws-mail.md]: Typo here

File: docs/claws-mail.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# vdirsyncer with Claws Mail
+
+First of all, you must know that you can **not** edit CardDav contacts only **watch** CardDav contacts with Claws-Mail
+
+## Preparation
+First of al you need to install vdirsyncer, for that look [here](https://vdirsyncer.pimutils.org/en/stable/installation.html).
```
Comment (untitaker) [inline docs/claws-mail.md]: You now wrote "first of all" twice (further down you use it once more), also there's a typo here ("al" vs "all")

File: docs/claws-mail.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+password = "PASSWORD"
+
+[pair contacts]
+a = "local"
+b = "online"
+collections = ["from a", "from b"]
+conflict_resolution = "b wins"
+```
+
+First of all in the general section, we define the status folder path, for some logging information from vdirsyncer.
```
Comment (untitaker) [inline docs/claws-mail.md]: It's technically not logging, but discovered collections and generally stuff that needs to persist between syncs.

File: docs/claws-mail.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+# vdirsyncer with Claws Mail
+
+First of all, you must know that you can **not** edit CardDav contacts only **watch** CardDav contacts with Claws-Mail
+
+## Preparation
+First of al you need to install vdirsyncer, for that look [here](https://vdirsyncer.pimutils.org/en/stable/installation.html).
+Then we need to create some folders:
+
+    mkdir -p ~/.vdirsyncer/status
+    mkdir ~/.contacts
```
Comment (untitaker) [inline docs/claws-mail.md]: I think it should be only necessary to create the `.vdirsyncer` folder, all other folders are created automatically (or rather, vdirsyncer asks).

File: docs/claws-mail.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    vdirsyncer sync
+    
+## Claws Mail
+Open Claws-Mail.
+Got to **Tools** => **Addressbook**.
+
+Click on **Addressbook** => **New vCard**.
+Choose a name for the book. 
+
+Then search for the for the vCard in the folder **~/.contacts/**.
```
Comment (untitaker) [inline docs/claws-mail.md]: Double "for the"

File: docs/claws-mail.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
+# vdirsyncer with Claws Mail
+
+First of all, you must know that you can **not** edit CardDav contacts only **watch** CardDav contacts with Claws-Mail
```
Comment (untitaker) [inline docs/claws-mail.md]: To stay consistent with other terminology I'd suggest "read-only" here.

**Comment by untitaker** [comment]
@beli3ver huh?

**Comment by beli3ver** [comment]
Don't know how to pull here the new changes so I create a new one :dancing_men:

**Comment by untitaker** [comment]
You need to push to the same branch (so just "git commit" and "git push")

On 17 January 2017 15:59:06 CET, Malte Kiefer <notifications@github.com> wrote:
>Don't know how to pull here the new changes so I create a new one
>:dancing_men: 
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/pimutils/vdirsyncer/pull/537#issuecomment-273191802

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Pull request closed without merging**